src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const expensesServices = require('../services/expenses.services');
+const userService = require('../services/users.services');
+const filterExpenses = require('../helpers/filterExpenses');
+const { validate } = require('../helpers/userValidation');
+
+const getAll = (req, res) => {
+  const allExpenses = expensesServices.findAll();
+  const { userId, categories, from, to } = req.query;
+  const filters = {
+    userId,
+    categories,
+    from,
+    to,
+  };
+  const filteredExpenses = filterExpenses(allExpenses, filters);
```

Comment:
`expensesServices.findAll` should accept filters and return only needed expenses